### DISCUSSION TITLE
Send request to the backend as post

### DISCUSSION THREAD

**Discussion by BergBenji** [inertiaui/modal#1] [Ideas]
First of all, i love the idea an the current result of your plugin!
It would be nice to send the request as a post to the backend. Currently it seems that only get requests possible.
Something like this:
```
<ModalLink :href="route('cont.modal', {'clientid':probs.clientid})" :method"'post'" :data="preparedDataVariable">
        Link Modal
</ModalLink>
```

**Comment by pascalbaljet** [comment]
Thanks! You're right; it's only GET now. I'll add this to make `ModalLink` more compatible with `Link` (see #2).

**Reply by pascalbaljet** [reply]
Added in v0.2.0: https://inertiaui.com/inertia-modal/docs/basic-usage#method-and-data
